src/Main.java

```diff
@@ -0,0 +1,39 @@
+import parsers.BookParser;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        //Create name of file path that will be correct for a specific operating system
```

Comment:
it is a bad practice to comment on all lines of code, code needs to be self-descriptive